runtime/doc/api-functions.txt

```diff
@@ -0,0 +1,722 @@
+*api-funcs.txt*         Neovim API Function Reference                   {Nvim}
+
+Note: This documentation is generated from Neovim's API source code.
+
+Contents:
+
+1. Buffer Functions   |nvim-api-buffer|
```

Comment:
I think order should be:

1. Vim (or Nvim or Global or whatever)
2. Buffer
3. Window
4. Tabpage
5. UI

Global functions should be first. Then the three proper object types should be in a row. UI methods works a bit different as they don't have an explicit reciever object, it is implicitly associated with the channel.